config/keybow/config.exs

```diff
@@ -10,15 +10,26 @@ config :shoehorn,
   init: [:nerves_runtime, :nerves_pack],
   app: Mix.Project.config()[:app]
 
+# Erlinit can be configured without a rootfs_overlay. See
+# https://github.com/nerves-project/erlinit/ for more information on
+# configuring erlinit.
+
+config :nerves,
+  erlinit: [
+    ctty: "tty1",
+    hostname_pattern: "nerves-%s",
```

Comment:
What is this for?  And don't we use `xebow.local` for our hostname?

Reply:
Not sure.  I think we were experimenting with how the erlinit config worked.  What's currently set matches the default from the system image so it can just be removed.